tests/GenerateTests.fs

```diff
@@ -605,71 +571,68 @@ module GenerateTests =
 
     try
 
-      Record structure.Name { yield! expandFields structure }
+      Record(structure.Name) {
+        // expandFields structure
+        Field("TODO", LongIdent "TODO")
```

Comment:
This was missing. I was able to fix it with this:

```fsharp
      Record(structure.Name) {
        yield! 
          expandFields structure
          |> List.map (fun (name, t) -> Field(name, t))
      }
```

Reply:
Nice 👍